pkg/controller/cainjector/setup.go

```diff
@@ -106,13 +137,7 @@ func RegisterCertificateBased(mgr ctrl.Manager) error {
 	sources := []caDataSource{
 		&certificateDataSource{client: mgr.GetClient()},
 	}
-	for _, setup := range injectorSetups {
-		if err := Register(mgr, setup, sources...); err != nil {
```

Comment:
How come we've moved the `for` loop into a new function `registerAllInjectors`, and removed the `Register` function altogether? It doesn't seem like this is a required change in order to add the AuditSink type?

Reply:
Hard to remember, but yes, it's not a required change. I beleive I couldn't think of why the register function should be exported so made it private, but that't not related to why the `registerAllInjectors ` was created. It was created to avoid duplicating the error handling and logging code in the loop to both places.

Reply:
@munnerz do you want the register method get recovered?

Reply:
@munnerz I've restored it, please take a look